README.md

```diff
@@ -0,0 +1,143 @@
+
+#### Easing
+The easing paramater defaults to a linear ease, but you can pass any function you want into it with the following signature
+`Func<float, float>`. This library comes with several built-in ones; the most common ones found [here](https://easings.net).
+
+They are all under the ease class with nested classes/functions for each type.
+ 
+ Example:
+Scale from 0f to 1f, over half a second using [back-out easing](https://easings.net/#easeOutBack)
+```c#
+transform.Scale(0f, 1f, 0.5f, Ease.Back.Out)
+```
+
+### Playing Tweens
+Once you have created one, playing a tween is easy
+```c#
+tween.Play();
+```
+
+You can also pass in a callback for when the tween completes
+```c#
+tween.Play(() => { Debug.Log("It's complete"); });
+```
+
+**Play(repeatCount)**
+
+### TweenCollections
+There are 2 types of animation collections, a `SequencedAnimation` for creating a sequence of tweens and `ParalleledAnimation` for playing tweens in parallel.
+
+Both are types of `AnimationCollection`.
+
+You can create either of these collections using a fluent interface.
+There are extension methods for every type of tween. As well as an `Add` function for adding any Tween.
+
+```c#
+var sequence = new SequencedAnimation()
+    // Use extension methods
+    .FadeTo(spriteRenderer, 1f)
+    .Scale(transform, 0, 1f, 0.24, Ease.Back.Out)
+    // Or use a paralleled within the sequence
+    .Parallel(p => p
+        // thse are nested within the parallel
+        .Move(transform, positionA, positionB)
+        .RotateY(transform, 0, 360))
+    // or just use add
+    .Add(new MoveAnimation(target, from, to, duration)
+```
+
+ParalleledAnimations are created in the same way.
+
+### Advanced playback features
+
+**Abort()**
+
+A tween can be aborted during playback by calling the abort function. The objects will be left in whatever state they are in when the tween is aborted.
+
+**FastForward()**
+
+A tween can also be fast forwarded to the end. This effectively immediately stops the tween and fires the final update frame, so the objects are left in a state equivelant to how they would be if the tween had completed.
+
+**Reverse()**
+
+Calling reverse will put the tween in reverse. It can be used prior to or during playback. If reversed it will cause the animation to play in reverse. For sequenced collections, it will play the animations in reverse order
+
+**ChangeSpeed(float multiplier)**
+
+Change speed will change the animations speed by a multiplier. For example `ChangeSpeed(2.0f)` will double the speed of the animation. The effects are permanent and therefore cumulative, so calling `ChangeSpeed(2.0f)` twice will result in the animation now being 4 times as fast as the original.
+
+**ScaleTime(float duration)**
+
+Scale time allows you to change the duration of an animation. This will end up changing the speed.
+
```

Comment:
Does multiple calls to ScaleTime have a accumulating effect, as ChangeSpeed does? If not it should say so on the document, so that it's used by developers instead of ChangeSpeed where appropriate.

Reply:
I don't believe so. I believe it will just calculate the speed modifier needed to change it to the requested duration. The 2nd call should calculate the same results. Not 100% sure though @Rollo-Liu would know more. If not we can read the code and find out.